<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<!--
SPDX-FileCopyrightText: 1999 David Faure <faure@kde.org>
SPDX-FileCopyrightText: 2004 Nicolas GOUTTE <goutte@kde.org>
SPDX-FileCopyrightText: 2024 Carl Schwan <carl@carlschwan.eu>
SPDX-License-Identifier: LGPL-2.0-or-later
-->
 <class>ExportDialogUI</class>
 <widget class="QDialog" name="ExportDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>334</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0">
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="encodingPage">
      <attribute name="title">
       <string>End of Line</string>
      </attribute>
      <layout class="QHBoxLayout" name="encodingLayout">
       <item>
        <layout class="QFormLayout">
         <item row="0" column="0">
          <widget class="QLabel">
           <property name="text">
            <string>End of Line:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioEndOfLineLF">
           <property name="text">
            <string>&amp;UNIX style (recommended; line feed only)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioEndOfLineCRLF">
           <property name="text">
            <string>&amp;Windows style (carriage return and line feed)</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="radioEndOfLineCR">
           <property name="text">
            <string>&amp;MacOS style (carriage return only)</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Sheets</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="2" column="0">
        <widget class="QListWidget" name="m_sheetList">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel4">
         <property name="text">
          <string>Select sheets to export:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_delimiterLineBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Delimiter Line</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="m_sheetDelimiter">
            <property name="text">
             <string>********&lt;SHEETNAME&gt;********</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;SHEETNAME&gt; gets replaced by the name of the next sheet.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="m_delimiterAboveAll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Print delimiter line above every sheet</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Cells</string>
      </attribute>
      <layout class="QHBoxLayout" name="cellLayout">
       <item>
        <layout class="QFormLayout">
         <item row="0" column="0">
          <widget class="QLabel">
           <property name="text">
            <string>Delimiter:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="m_radioComma">
           <property name="text">
            <string>Comma</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="m_radioSemicolon">
           <property name="text">
            <string>Semicolon</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="m_radioTab">
           <property name="text">
            <string>Tabulator</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="m_radioSpace">
           <property name="text">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout">
           <item column="0">
            <widget class="QRadioButton" name="m_radioOther">
             <property name="text">
              <string>Other:</string>
             </property>
            </widget>
           </item>
           <item column="1">
            <widget class="QLineEdit" name="m_delimiterEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>10</height>
              </size>
             </property>
             <property name="maxLength">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel">
           <property name="text">
            <string>Quotes:</string>
           </property>
           <property name="buddy">
            <cstring>m_comboQuote</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="m_comboQuote">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>&quot;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>'</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1">
    <widget class="QFrame">
     <property name="frameStyle">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="fixedHeight">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2">
    <layout class="QVBoxLayout" name="bottomLayout">
     <item row="0">
      <widget class="QCheckBox" name="m_selectionOnly">
       <property name="text">
        <string>Export selection only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3">
    <widget class="QDialogButtonBox" name="m_buttonBox">
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>KComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
